teemaderegister-be/src/routes.js

```diff
@@ -41,12 +42,18 @@ router.get('/users/profile', jwtEnsure, asyncMiddleware(users.getProfile))
 router.put('/users/profile', jwtEnsure, validate.userAccountUpdate, asyncMiddleware(users.updateUser))
 router.put('/users/password', jwtEnsure, validate.userPasswordUpdate, asyncMiddleware(users.updatePassword))
 router.post('/users/upload-picture', jwtEnsure, multerMiddleware('profileImage'), asyncMiddleware(users.uploadPicture))
+
+router.post('/factor', jwtEnsure, asyncMiddleware(factor.create))
```

Comment:
would prefix them still with auth/local... and then add /factor. As they are for local auth

Reply:
Done setup now  with auth/local